src/windowsdesktop/src/sfx/Directory.Build.targets

```diff
@@ -52,10 +52,14 @@
     <PropertyGroup>
       <PackageOverridesInputPath>$(MSBuildThisFileDirectory)PackageOverrides.txt</PackageOverridesInputPath>
       <PackageOverridesOutputPath>$(BaseOutputPath)PackageOverrides.txt</PackageOverridesOutputPath>
+
+      <!-- Add the RuntimeWindowsDesktopPackageLibrary item information from the Microsoft.Internal.Runtime.WindowsDesktop.Transport package. -->
+      <RuntimeWindowsDesktopPackageLibraries>@(RuntimeWindowsDesktopPackageLibrary->'%(PackageId)|%(PackageVersion)', '
+')</RuntimeWindowsDesktopPackageLibraries>
     </PropertyGroup>
 
     <ItemGroup>
-      <CreatePackageOverridesTemplateProperty Include="ProductVersion=$(Version)" />
+      <CreatePackageOverridesTemplateProperty Include="RuntimeWindowsDesktopPackageLibraries=$(RuntimeWindowsDesktopPackageLibraries)" />
     </ItemGroup>
 
     <GenerateFileFromTemplate
```

Comment:
This can be replaced with `WriteLinesToFile` now.

Reply:
The `GenerateFileFromTemplate` task provides more flexibility in case we ever want to hardcode values in there or use some other layout. I would rather keep it.